mslib/mscolab/conf.py

```diff
@@ -68,7 +68,10 @@ class default_mscolab_settings:
 
     # mscolab file upload settings
     UPLOAD_FOLDER = os.path.join(DATA_DIR, 'uploads')
-    MAX_UPLOAD_SIZE = 2 * 1024 * 1024  # 2MB
+    MAX_UPLOAD_SIZE = 2 * 1024 * 1024  # 2MiB
+
```

Comment:
I assume this shoul be used in https://github.com/Open-MSS/MSS/blob/develop/mslib/mscolab/app/__init__.py